client/packages/programs/src/api/hooks/useProgramList.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ProgramSortFieldInput,
+  useQuery,
+} from '@openmsupply-client/common';
+import { ProgramFragment } from '../operations.generated';
+import { useProgramsGraphQL } from '../useProgramsGraphQL';
+import { LIST, PROGRAM } from './keys';
+
+export const useProgramList = () => {
+  const { api, storeId } = useProgramsGraphQL();
+
+
+  const queryKey = [PROGRAM, LIST];
+  const queryFn = async (): Promise<{
+    nodes: ProgramFragment[];
+    totalCount: number;
+  }> => {
+    const filter = {
+      isImmunisation: false,
```

Comment:
Currently this filter still returns `missing_program` - where's the best place to filter that out do you think?

Reply:
I think create_filtered_query should filter it out, just like `is_deleted`, in fact i'd be tempted to add `is_deleted` to that table and then mark `missing_program` as deleted.
It's only used during sync translation from what I can tell.